gov2/dynamodb/scenarios/scenario_partiql_batch_test.go

```
@@ -0,0 +1,103 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Unit tests for the PartiQL batch scenario.
+
+package scenarios
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"github.com/awsdocs/aws-doc-sdk-examples/gov2/dynamodb/actions"
+	"github.com/awsdocs/aws-doc-sdk-examples/gov2/dynamodb/stubs"
+	"github.com/awsdocs/aws-doc-sdk-examples/gov2/testtools"
+)
+
+// PartiQLBatchScenarioTest encapsulates data for a scenario test.
+type PartiQLBatchScenarioTest struct {
+	TableName string
+}
+
+// TestRunPartiQLBatchScenario runs the scenario multiple times. The first time, it
+// runs with no errors. In subsequent runs, it specifies that each stub in the sequence
+// should raise an error and verifies the results.
```

should raise an error and verifies the results. --> should raise an error, and it verifies the results.

---

fixed